README.md

```diff
@@ -7,1334 +7,81 @@
 [![Downloads/week](https://img.shields.io/npm/dw/fauna.svg)](https://npmjs.org/package/fauna)
 [![License](https://img.shields.io/npm/l/fauna.svg)](https://github.com/fauna/fauna/blob/master/package.json) -->
 
-This tools gives you access to [Fauna](http://fauna.com/) directly from your CLI.
+The Fauna CLI lets you access [Fauna](http://fauna.com/) from your terminal. You
+can use the CLI to:
 
-It also includes a [Shell](#shell) so you can issue queries to Fauna without
-needing to install additional libraries.
+* Log in to your Fauna account.
+* Create and manage Fauna
+  [databases](https://docs.fauna.com/fauna/current/learn/data-model/databases/)
+  and [keys](https://docs.fauna.com/fauna/current/learn/security/keys/).
+* Manage [database schema](https://docs.fauna.com/fauna/current/learn/schema/)
+  as `.fsl` files on your local machine.
+* Run [FQL queries](https://docs.fauna.com/fauna/current/learn/query/) from
+  [files](https://docs.fauna.com/fauna/current/build/cli/commands/eval/) or in
+  an [interactive
+  shell](https://docs.fauna.com/fauna/current/build/cli/commands/shell/).
 
-You can install it via npm like this:
+![fauna-cli](https://github.com/user-attachments/assets/2f7d53f5-f445-4cb1-b90c-455f3a0e51a8)
 
-```sh
-$ npm install -g fauna-shell
-```
-
-You can upgrade like this:
-
-```
-$ npm update -g fauna-shell
-```
-
-<!-- toc -->
-* [Fauna CLI](#fauna-cli)
-* [Usage](#usage)
-* [Technical Requirements](#technical-requirements)
-* [Shell](#shell)
-* [Connecting to different endpoints](#connecting-to-different-endpoints)
-* [Connecting to local endpoints](#connecting-to-local-endpoints)
-* [Overriding Connection Parameters](#overriding-connection-parameters)
-* [Executing queries from a file](#executing-queries-from-a-file)
-* [List of Commands](#list-of-commands)
-* [Development](#development)
-<!-- tocstop -->
-
-# Usage
-
-The **Fauna CLI** allows you to do issue queries, modify database schema, and
-create keys and databases.
-
-First lets configure our connection to a Fauna account. If you don't have an
-account, you can create a free one [here](https://dashboard.fauna.com).
-
-To log in, run the following command:
-
-```sh
-$ fauna cloud-login
-```
+## Requirements
 
-You will be prompted for your `email` and `password` from your
-[Fauna](https://dashboard.fauna.com) account.
+[Node.js](https://nodejs.org/en/download/package-manager) v20.x or later.
```

Comment:
I believe we support v18.x right now (see the [package.json's "engines" field](https://github.com/fauna/fauna-shell/blob/main/package.json#L64) for what we declare to the package manager, and our [.nvmrc](https://github.com/fauna/fauna-shell/blob/main/.nvmrc) for what we declare to developers making changes to the CLI).

Reply:
Good looking out. I blindly carried this over from the current README. Updated.